tools/build_release_uvision.bat

```diff
@@ -15,22 +15,40 @@
 :: See the License for the specific language governing permissions and
 :: limitations under the License.
 ::
+
+@if exist .gitignore goto label0
+	@echo Error: working directory must be root of DAPLink repository
+	@exit /B 1
+
+:label0
+
 rmdir /q /s uvision_release
+@if %errorlevel% neq 0 exit /B %errorlevel%
+
 mkdir uvision_release
+@if %errorlevel% neq 0 exit /B %errorlevel%
+
+@git --version 2> nul
+@if %errorlevel% neq 0 echo Error: git must be in PATH && exit /B %errorlevel%
+
 git rev-parse --verify HEAD                     > uvision_release\git_info.txt
 git diff --no-ext-diff --quiet --exit-code      >> uvision_release\git_info.txt
+
 echo Uncommitted Changes: %errorlevel%          >> uvision_release\git_info.txt
+
 virtualenv env
 call env\Scripts\activate
-REM use project requirements if not specified
+
+@REM use project requirements if not specified
 if [%1]==[] pip install -r requirements.txt
-REM use custom requirements if specified
+@REM use custom requirements if specified
 if not [%1]==[] pip install -r %1
 pip freeze                                      > uvision_release\build_requirements.txt
-REM build but continue if there are errors
+
+@REM build but continue if there are errors
 progen generate -t uvision -b
-SET LEVEL=%ERRORLEVEL%
+@SET level=%errorlevel%
```

Comment:
It was not causing a problem. But the scripts were inconsistent and I find
that distracting, and I tried to bring consistency at least to the scripts
I was touching. Uppercase vs lowercase--I don't really have a preference. I
arbitrarily picked one and I can easily switch to the other and make it
consistent if you like

On Wed, Apr 13, 2016 at 8:46 AM, Russ Butler <notifications@github.com>
wrote:

> In tools/build_release_uvision.bat
> <https://github.com/mbedmicro/DAPLink/pull/88#discussion_r59549882>:
>
> >  progen generate -t uvision -b
> > -SET LEVEL=%ERRORLEVEL%
> > +@SET level=%errorlevel%
>
> Was caps causing problems, or is this just a cosmetic change?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mbedmicro/DAPLink/pull/88/files/113397cce1f2bcecff59bbec1772e5ff32b9f777#r59549882>
>